tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -72,6 +82,23 @@ def test_plot_pareto_front_2d(
         assert figure.get_xlabel() == target_names[axis_order[0]]
         assert figure.get_ylabel() == target_names[axis_order[1]]
 
+    if axis_order is not None:
+        pareto_front_points = np.array([[1.0, 0.0], [0.0, 1.0]])[:, axis_order]
+    else:
+        pareto_front_points = np.array([[1.0, 0.0], [0.0, 1.0]])
+    assert pareto_front_points.shape == (2, 2)
+
+    path_offsets = list(map(lambda pc: pc.get_offsets(), figure.findobj(PathCollection)))
+    exists_pareto_front = any(
+        map(lambda po: allclose_as_set(po, pareto_front_points), path_offsets)
+    )
+    exists_dominated_trials = any(
+        map(lambda po: allclose_as_set(po, np.array([[1.0, 1.0]])), path_offsets)
+    )
+    assert exists_pareto_front
+    if include_dominated_trials:
+        assert exists_dominated_trials
```

Comment:
How about testing not only for `Test with `target_names` argument` but also `Test with three trials`?

Reply:
Test `three trials` also in  https://github.com/optuna/optuna/pull/3183/commits/aed169559609ea1d74c585aa8d4bca70d771cad6.